src/app/containers/Metadata/index.jsx

```diff
@@ -50,6 +50,14 @@ const getTimeTags = (timeTag, assetType) => {
   return new Date(timeTag).toISOString();
 };
 
+const getAppleTouchUrl = service => {
```

Comment:
Hey, for this could we maybe go for something like
```

Reply:
done 👍